### ISSUE TITLE
FNX-5316 ⁃ [Bug] Quick action bar clipped

### ISSUE THREAD

**Issue by SeburoSUMO** [mozilla-mobile/fenix#1892]
## Steps to reproduce
Open the app from first run (no history or settings changed).
Search for URL
Open page

### Expected behavior
Bookmark and share icons are shown fully

### Actual behavior
Bookmark and share icons overlapped by address bar (image attached to illustrate)
![Screenshot](https://user-images.githubusercontent.com/31939113/56516251-daca3a80-6531-11e9-9206-debfc08838ca.jpg)UNITO-UNDERSCORE!1555954874964!

### Device information
* Android device: Moto E5, Android 8.0.0
* Fenix version: 1.0.1917 (Build #11121209 GV 68.0a1-20190417093858)

**Comment by ekager** [comment]
Do you mean that you can't pull up this menu to show the entirety of these buttons? This sheet is supposed to be displayed behind the toolbar and be pulled up and down.

**Comment by ekager** [comment]
And on first run, this sheet is supposed to show a "bounce" animation that reveals it just a bit before sliding back behind the toolbar

**Comment by lime124** [comment]
@SeburoSUMO can you clarify when this is happening - the steps to reproduce? it's unclear if this is a usability issue we need to comment on to fix or it's just being buggy. thanks!

**Comment by SeburoSUMO** [comment]
I opened the app at a first run state, entered "mozilla.org" and the page opens as displayed above with the bottom of the bookmark and share icons being clipped by the address bar.  Swiping up displayed the icons fully and made the address bar close.  However...

Since I reported that, I have updated to  Fenix version: 1.0.1917 (Build #11141216 GV 68.0a1-20190422094240).  Repeating the same steps leads to:

- Icons displaying fully on first run.
- Swiping up displays the icon names.

Looks like this issue is resolved.  The behavior I am seeing now I would have no reason to flag.
